build.gradle

```diff
@@ -329,6 +329,33 @@ testClusters.integTest {
         setting("plugins.security.authcz.admin_dn", "\n- CN=kirk,OU=client,O=client,L=test, C=de")
         setting('plugins.security.restapi.roles_enabled', '["all_access", "security_rest_api_access"]')
         setting('plugins.security.system_indices.enabled', "true")
+        setting('plugins.security.system_indices.indices', '[' +
```

Comment:
I'm not sure how adding this list here relates to limiting what we delete in the test case class.   We're mostly adding stuff that we will ignore, and a bunch of indices unrelated to our tests that won't change.

If we need this set, Is there a way to fetch this programmatically?  Copy-pasting [a list from another plugin](https://github.com/opensearch-project/security/blob/2dab119387212d5047062100f799934b7ffbffea/tools/install_demo_configuration.sh#L388C3-L388C62) that may change in the future is brittle.   It looks like that script adds them to the `opensearch.yml` file can we read them from there?  Or better yet at run-time from the settings?

If we keep this as-is, please add a comment saying where these come from so future-you and future-me know when a test breaks where to re-sync this list.

Reply:
This list comes from the security demo configuration [here](https://github.com/opensearch-project/security/blob/2dab119387212d5047062100f799934b7ffbffea/tools/install_demo_configuration.sh#L388) which is applied to a security enabled integration test cluster. We have to manually set this setting since we're not using the demo script.

Reply:
Can we try something like CCR plugin [here](https://github.com/opensearch-project/cross-cluster-replication/blob/3ea239b5b308b8c053511fb2da79ac37f12b2340/build.gradle#L685) to run the demo script? Otherwise, we would need to keep the track of the system indices in the demo file and update our build.gradle.

Reply:
CCR applies the security config in the same way we are [here](https://github.com/opensearch-project/cross-cluster-replication/blob/3ea239b5b308b8c053511fb2da79ac37f12b2340/build.gradle#L573), though they have omitted protected indices other than their own.

Reply:
Yes, that's what we were trying to do with just our indices and ml-commons. I am not sure if adding other plugins indices would be the right way to go around here.

Reply:
Ill remove them